gen/shuffleboard/ShuffleboardContainer.yml

```diff
@@ -6,66 +6,72 @@ extra_includes:
 - frc/shuffleboard/ShuffleboardLayout.h
 - frc/shuffleboard/SimpleWidget.h
 
-functions:
-  Add:
-    overloads:
-      cs::VideoSource&:
-      wpi::Twine&, cs::VideoSource&:
 classes:
   ShuffleboardContainer:
-    shared_ptr: false
+    # virtual base issue: robotpy-build/166
+    force_no_trampoline: true
     attributes:
       m_isLayout:
     methods:
       ShuffleboardContainer:
-        ignore: true
       GetComponents:
-        # https://github.com/pybind/pybind11/issues/263
-        # .. use custom code here?
-        ignore: true
       GetLayout:
         overloads:
-          wpi::Twine&, BuiltInLayouts:
-          wpi::Twine&, LayoutType&:
-          wpi::Twine&, wpi::Twine&:
-          wpi::Twine&:
+          std::string_view, BuiltInLayouts:
+          std::string_view, LayoutType&:
+          std::string_view, std::string_view:
+          std::string_view:
       Add:
         overloads:
-          wpi::Twine&, frc::Sendable&:
-            keepalive:
-            - [1, 3]
-          wpi::Twine&, cs::VideoSource&:
+          std::string_view, wpi::Sendable&:
+          std::string_view, cs::VideoSource&:
             ignore: true
-          frc::Sendable&:
-            keepalive:
-            - [1, 2]
+          wpi::Sendable&:
           cs::VideoSource&:
             ignore: true
-          wpi::Twine&, std::shared_ptr<nt::Value>:
-          wpi::Twine&, bool:
-          wpi::Twine&, double:
-          wpi::Twine&, int:
-          wpi::Twine&, wpi::Twine&:
-          wpi::Twine&, char*:
-          wpi::Twine&, wpi::ArrayRef<bool>:
-          wpi::Twine&, wpi::ArrayRef<double>:
-          wpi::Twine&, wpi::ArrayRef<std::string>:
+          std::string_view, nt::Value&:
+          std::string_view, bool:
+          std::string_view, double:
+          std::string_view, float:
+          std::string_view, int:
+          std::string_view, std::string_view:
+          std::string_view, char*:
+            ignore: true
+          std::string_view, std::span<const bool>:
+          std::string_view, std::span<const double>:
+          std::string_view, std::span<const float>:
+          std::string_view, std::span<const int64_t>:
+          std::string_view, std::span<const std::string>:
+      AddCamera:
+        ignore: true
```

Comment:
Is there a reason this is ignored?

Reply:
You can't have cscore loaded in the same process as everything else otherwise bad things happen

Reply:
This method shouldn't depend on cscore?

Reply:
Oh, you're right. Oops.

... all of shuffleboard is disabled at the moment since I didn't have time to re-enable it. It might work?

Reply:
Right, fair enough. That means #691 was prematurely closed, right?